addons/udes_stock/models/stock_picking.py

```diff
@@ -681,3 +682,57 @@ def _unassign_users(self, skip_users=None):
         users_to_unassign.sudo().write(
             {"u_picking_id": False, "u_picking_assigned_time": False}
         )
+
+    @staticmethod
+    def get_stock_investigation_message(quants):
+        """
+        Dummy method to be overridden on a per-customer basis.
+
+        :returns: str
+        """
+        return ""
```

Comment:
Can we add a default message like: product x quantity?

Reply:
Just seen that we have `if picking_details:`, so no need for this, will resolve.